css/style.css

```diff
@@ -4,6 +4,24 @@
  *
  */
  @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+ @import url('https://fonts.googleapis.com/css2?family=Oxygen&display=swap');
+ @import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@500&display=swap');
+
+body {
+  background-color: #f8f9d2;
+  background-image: linear-gradient(315deg, #f8f9d2 0%, #e8dbfc 74%);

```

Comment:
I don't think that we need a gradient background. 🤔

Reply:
Sure.